src/index.js

```diff
@@ -0,0 +1,162 @@
+'use strict';
+
+function Modal(clickElm, modalParent, options) {
+  options = options || {};
+
+  var that = this;
+  clickElm.addEventListener('click', function(e) {
+    e.preventDefault();
+    that.show();
+  }, false);
+  this.clickElm = clickElm;
+  this.modalParent = modalParent || document.documentElement;
+  // Create a unique instance id.
+  this.id = '_' + Math.random().toString(36).substr(2, 9);
+  this.modalWidth = options.modalWidth || 300;
+  this.modalHeight = options.modalHeight || 500;
+  this.closeDelayMs = typeof options.closeDelayMs === 'number' ?
+                        options.closeDelayMs : 300;
+  return this;
+}
+
+Modal.prototype = {
+  iframeSrc: 'http://pay.dev:8000/',
```

Comment:
Yep was thinking that down the road we should probably have this configurable but only accept certain values.